docs/imaging_model/README.md

```diff
@@ -0,0 +1,64 @@
+# Imaging and Report Model
+
+## Motivation
+
+Participants often have files in addition to genomic files ranging from
+MRI images, histology images, and reports.
+Some of these file types relate directly to the biospecimen rather than the
+participant, eg: proteomics and histology.
+In order to capture and distribute these files, we need to model them
+appropriately.
+
+## Implementation
+
+Files will need to be distributed through gen3, so the models will need to
+inherit the Indexd model similar to GenomicFiles.
+Each new file type will have a unique prefix for the model as well as specific
+metadata fields unique to that type of file.
+
+![Imaging Model](ImagingModelTables.png)
+
+### Radiology File and Report
+
+RadiologyFile and RadiologyReport have a many-to-many relationship to
+each-other and one-to-many from Participant.
+When linking a file with a report, we must ensure that they both share the
+same participant. This is similar to what we do when linking biospecimens and
```

Comment:
Is it true that every RadiologyReport must have one or more RadiologyFiles related to it? If so, it should be better to remove the relationship between RadiologyReport and Participant because this relationship can be derived from relationship between RadiologyFile and Participant, hence is redundant.

Also, the many-to-many relationship between RadiologyFile and RadiologyReport will need to be modelled using a middle table that has one-to-many relationships to both RadiologyFile and RadiologyReport. Such middle table may include three columns as composite key, something like below:

| participant_id | radiology_file_id | radiology_report_id |

This will at db level guarantee `When linking a file with a report, we must ensure that they both share the
same participant.`

Same comment for `HistologyImage` and `PathologyReport` tables below.